src/products/utils/handlers.ts

```diff
@@ -120,6 +124,35 @@ export function createAttributeMultiChangeHandler(
   };
 }
 
+export function createAttributeFileChangeHandler(
+  changeAttributeData: FormsetChange<string[]>,
+  attributesWithNewFileValue: FormsetData<FormsetData<null, File>>,
+  addAttributeNewFileValue: (data: FormsetAtomicData<null, File>) => void,
+  removeAttributeNewFileValue: (id: string) => void,
+  triggerChange: () => void
+): FormsetChange<File> {
+  return (attributeId: string, value: File) => {
+    triggerChange();
+    if (value) {
+      addAttributeNewFileValue({
+        data: null,
+        id: attributeId,
+        label: null,
+        value
+      });
+    } else {
+      const removeingNewFileValue = attributesWithNewFileValue.find(
```

Comment:
Typo. Please install spellcheck in your editor. There are a few to choose if you're using VSCode.